src/envo/envo-edit.owl

```diff
@@ -39106,6 +39194,75 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03510041>
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03510041> "wood stain"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03510041> <http://purl.obolibrary.org/obo/ENVO_03510040>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520000> (low atmospheric level)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_03520000> "To be revised when we have information about the real definition. The definitions given by WMO cloud atlas are entered as comment, as these are not the defining features of the levels but the result of whatever it is that defines the levels (I assume temperature, density, etc. but have not yet found clear information about this)."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03520000> <http://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_03520000> "low étage"@en)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03520000> "A part of the troposphere which extends from the Earth's surface to 2 km (0 - 6500 ft) height. [https://cloudatlas.wmo.int/en/useful-concepts.html#levels]"@en)
```

Comment:
Here and in similar additions, add the URL/URI/IRI as an AP on the comment/def/etc

Reply:
Done 👍